### PR TITLE
Adding missing stubs

### PR DESCRIPTION
**Pull request by atomflunder**
As discussed in #25 . I hope this looks and works alright.

### CODE REVIEW

**Comment by maxbachmann** [comment]
The last two arguments are keyword only:
```
def distance(
    s1: Sequence[Hashable],
    s2: Sequence[Hashable],
    *,
    processor: Optional[Callable] = None,
    score_cutoff: Optional[float] = None,
) -> int: ...
```

File: src/Levenshtein/__init__.pyi [hunk 1]
```diff
@@ -58,3 +52,38 @@ def setratio(
...
+    *,
+    processor: Optional[Callable] = None,
+    score_cutoff: Optional[float] = None,
+) -> float: ...
+def jaro_winkler(
+    s1: Sequence[Hashable],
+    s2: Sequence[Hashable],
+    *,
+    processor: Optional[Callable] = None,
+    score_cutoff: Optional[float] = None,
```
Comment (maxbachmann) [inline src/Levenshtein/__init__.pyi]:
```suggestion
def jaro_winkler(
    s1: Sequence[Hashable],
    s2: Sequence[Hashable],
    *,
    prefix_weight: float = 0.1,
    processor: Optional[Callable] = None,
    score_cutoff: Optional[float] = None,
```

File: src/Levenshtein/__init__.pyi [hunk 2]
```diff
@@ -58,3 +52,38 @@ def setratio(
 def seqratio(
     strlist1: List[Union[str, bytes]], strlist2: List[Union[str, bytes]]
 ) -> float: ...
+def distance(
+    s1: Sequence[Hashable],
+    s2: Sequence[Hashable],
+    *,
+    processor: Optional[Callable] = None,
+    score_cutoff: Optional[float] = None,
+) -> int: ...
```
Comment (maxbachmann) [inline src/Levenshtein/__init__.pyi:62]:
```suggestion
def distance(
    s1: Sequence[Hashable],
    s2: Sequence[Hashable],
    *,
    weights: Optional[Tuple[int, int, int]] = (1, 1, 1),
    processor: Optional[Callable] = None,
    score_cutoff: Optional[float] = None,
) -> int: ...
```

**Comment by maxbachmann** [comment]
ratio does not need the additional argument

**Comment by atomflunder** [comment]
Oops yes, sorry about that, should not code this tired.

**Comment by maxbachmann** [comment]
Thanks

**Comment by atomflunder** [comment]
Thanks for your assistance!

**Pull request merged**